tools/grecorder/grecorder.c

```diff
@@ -23,10 +23,11 @@
 #include <semaphore.h>
 #include <pthread.h>
 #include <signal.h>
+#include <unistd.h> /* sleep */
 #include <getopt.h>
 #define __USE_LARGEFILE64
 #include <sys/statvfs.h>
-
+#include <glib/gprintf.h> /* g_print */
```

Comment:
suggest to use printf instead of g_printf

Reply:
OK,  but g_print() can be redirected to another function using a function pointer and maybe its consistent with rest of usage in gstreamer as well.